lib/instrumentation/index.js

```diff
@@ -69,19 +70,25 @@ Instrumentation.prototype.start = function () {
 Instrumentation.prototype.addEndedTransaction = function (transaction) {
   if (this._started) {
     var queue = this._queue
+
     debug('adding transaction to queue %o', {id: transaction.id})
+
+    var payload = new AsyncValuePromise()
+
+    payload.then(null, function (err) {
```

Comment:
I also added the `catch` method in 1.1.0, by the way. ;)

Reply:
Refactored in #288